packages/create-react-app/index.js

```diff
@@ -111,20 +147,22 @@ function createApp(name, verbose, version, template) {
   checkAppName(appName);
   fs.ensureDirSync(name);
   if (!isSafeToCreateProjectIn(root)) {
-    console.log('The directory ' + chalk.green(name) + ' contains files that could conflict.');
+    console.log(
+      'The directory ' +
+        chalk.green(name) +
+        ' contains files that could conflict.'
```

Comment:
Still haven't quite figured out the best way to print those. Right now it's breaking on every + the same way && does. I've been meaning to try and try to fit as many as possible on the same line but haven't gotten around to.

cc @jlongster